probe-rs/src/debug/unit_info.rs

```diff
@@ -1000,37 +986,52 @@ impl UnitInfo {
                 child_variable.type_name = VariableType::Pointer(type_name);
                 self.process_memory_location(
                     debug_info,
-                    node.entry(),
+                    node,
                     parent_variable,
                     child_variable,
                     memory,
                     frame_info,
+                    // NOTE: surprisingly, as opposed to `void*`, this can be a `const void*`.
```

Comment:
Why did you move this comment? `const void*` vs `void*` is a weirdness where one has a `DW_AT_type`, and the other doesn't, that's why the two comments were placed in the two arms of the match.

Reply:
Don't think I did this on purpose, I'll move it back.

(I've tried out https://gitbutler.com/ recently, it's very promising but is still very much beta, it sometimes moves around changes...)